drydock/templates/drydock/k8s/jobs.yml

```diff
@@ -0,0 +1,113 @@
+{% for job in get_init_tasks() %}
+{{ job }}
+---
+{% endfor %}
+{% if DRYDOCK_MIGRATE_FROM %}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: drydock-upgrade-lms-job
+  labels:
+    drydock.io/component: job
+    drydock.io/target-service: lms
+    drydock.io/runner-service: lms
+  annotations:
+    argocd.argoproj.io/sync-wave: "{{ get_sync_waves_for_resource('drydock-upgrade-lms-job') }}"
+    argocd.argoproj.io/hook: Sync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  ttlSecondsAfterFinished: 100
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: lms
+        image: {{ DOCKER_IMAGE_OPENEDX }}
+        command:
+          - /bin/sh
+          - -c
+          - -e
+        args:
+        - |
+          {%- if DRYDOCK_MIGRATE_FROM == 13 %}
+          ./manage.py lms backpopulate_user_tours
+          {%- endif %}
+          {%- if DRYDOCK_MIGRATE_FROM in (13,14) %}
```

Comment:
to migrate from versions higher than 13 and 14 of open edx, is there any additional execution required?

Reply:
No sr. you can look at which commands we are using in [tutor upgrade command](https://github.com/overhangio/tutor/blob/master/tutor/commands/upgrade/k8s.py#L87)